src/api/assetDetails.ts

```diff
@@ -178,13 +183,13 @@ export const getAssetsDetails = (api: ApiPromise) => async () => {
   }
 
   const assets = rawAssetsData.map(([key, data]) => {
+    const id = key.args[0].toString()
+
     const { symbol = "N/A", decimals } =
-      assetsMeta.find(
-        (assetMeta) => assetMeta.id.toString() === key.args[0].toString(),
-      ) || {}
+      assetsMeta.find((assetMeta) => assetMeta.id.toString() === id) || {}
```

Comment:
It think we should be using descriptive names over variable shadowing. Any thoughts?

Reply:
what do you mean by this?

Reply:
In the scope of `find`, `assetMeta` is shadowed by the same variable name which could be considered as an anti-pattern. 

Can be solved by choosing different name for each element, ex: 

```
assetsMeta.find((assetMetaItem) => assetMetaItem.id.toString() === id) || {}
```

Reply:
God dammit, i take it all back :D misread the `s`. Sorry